### DISCUSSION TITLE
I can't use Tabular Editors

### DISCUSSION THREAD

**Discussion by LyleLiu666** [requarks/wiki#6183] [Help / Questions] [answered]
I have installed wiki with Docker,the version is  2.5.297

![image](https://user-images.githubusercontent.com/31182860/220035962-b2d3751e-4d7e-47e8-ade8-c1198b176125.png)
 
but I can't find the "Tabular " and "Blog" as document https://js.wiki/modules#editors

what I can see as follows
![image](https://user-images.githubusercontent.com/31182860/220036241-cd0508a7-f0c5-427d-aa7a-57dd4998c31e.png)

what should I do ？please help me！

**Comment by NGPixel** [comment, marked as answer]
As indicated in the list you linked to, the tabular and blog editors are not yet available and are planned for a future update.

**Reply by LyleLiu666** [reply]
thanks for this

Hope coming soon，excel and mind map both are what  I am looking forward to
